.github/workflows/examples.yml

```diff
@@ -37,8 +37,8 @@ jobs:
     env:
       OE_LICENSE: ${{ github.workspace }}/oe_license.txt
       PACKAGE: openff
-      PYTEST_ARGS: -r fE -v -x --tb=short -nauto
-      NB_ARGS: --nbval-lax --dist loadscope --ignore=examples/deprecated
+      PYTEST_ARGS: -r fE -v -x --tb=short -nauto --durations=10
```

Comment:
The examples workflow is slow. I don't want to fix that here, but
* seeing the timings now makes it easier to see what's slow
* seeing the timings in general helps safeguard against regressions